kit/src/main/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonWindowController.java

```diff
@@ -122,13 +124,18 @@ public void openWindow() {
     @Override
     protected void controllerDidLoadFxml() {
         super.controllerDidLoadFxml();
+        assert languageChoiceBox != null;
         assert commentCheckBox != null;
         assert formatCheckBox != null;
         assert textArea != null;
 
-        commentCheckBox.selectedProperty().addListener((ChangeListener<Boolean>) (ov, t, t1) -> update());
+        languageChoiceBox.getItems().addAll(SkeletonSettings.LANGUAGE.values());
+        languageChoiceBox.getSelectionModel().select(SkeletonSettings.LANGUAGE.JAVA);
 
-        formatCheckBox.selectedProperty().addListener((ChangeListener<Boolean>) (ov, t, t1) -> update());
+        ChangeListener<Object> updateOnChangeListener = (ov, t, t1) -> update();
```

Comment:
`updateOnChangeListener` could be replaced by `fxomDocumentRevisionListener`

Reply:
Fixed. Had to change the generic on `fxomDocumentRevisionListener` from `Number` to `Object`.